ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.js

```diff
@@ -85,6 +101,22 @@ function ClustersTablePanel({ selectedClusterId, setSelectedClusterId, searchOpt
         </div>
     ));
 
+    const installMenuOptions = [
+        <DropdownItem
+            key="link"
+            isExternalLink
+            description="Cluster installation guides"
+            href="https://docs.openshift.com/acs/installing/acs-installation-platforms.html"
```

Comment:
Van, can you please adapt according to pattern just merged in #4494
* conditional rendering according to `version`
* call `getVersionedDocs` function

Here is an example for a docs link elsewhere in Clusters:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Clusters/Components/CredentialExpiration.tsx#L86-L99

I verified that the versioned docs link exists for 3.73

Brad or I are available for a short call to collaborate, if needed.